cal/cal/helpers.py

```diff
@@ -66,7 +66,7 @@ def handle_time_string(time_str, timezone_str):
     return time
 
 
-def get_time_series(model, timezone='UTC', time_step='weekly', calendar_ids=None, start=None, end=None):
+def get_time_series(model, timezone='UTC', time_step='Week', calendar_ids=None, start=None, end=None):
```

Comment:
Ok I mean looks fine to me. @MaxFangX I think it should be fine, sometimes simplicity is better than PEP 8 :3